serialization-avro/src/main/java/io/axoniq/dev/samples/serializationavro/api/GiftCardBusinessErrorCode.java

```diff
@@ -0,0 +1,12 @@
+package io.axoniq.dev.samples.serializationavro.api;
+
+/**
+ * Public domain specific error codes
+ *
+ * @author Stefan Andjelkovic
+ */
+public enum GiftCardBusinessErrorCode {
```

Comment:
If we drop the `ExceptionWrappingHandlerInterceptor`, I think we can drop this as well.

Reply:
Yes